util/c_function.py

```
@@ -28,13 +31,14 @@ class ParsecFunction:
     arg_types: list[str] = field(default_factory=list)
     enums: list[Any] = field(default_factory=list)
     callee_names: list[str] = field(default_factory=list)
+    # MDE: Minor: I would name this `global_vars` rather than `llvm_globals`, which is not as clear.
     llvm_globals: list[Any] = field(
         default_factory=list
     )  # Cannot call this `globals` as it is a Python keyword.
     structs: list[Any] = field(default_factory=list)
 
     def __init__(self, raw_analysis: dict[str, Any]):
```

Ruff (0.14.8)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)